script/grant_revoke_gem_authority.rb

```diff
@@ -1,166 +1,142 @@
 # frozen_string_literal: true
 
+# The purpose of this script is to manage the authors of the rubygems
+# managed by the Samvera community.
+#
 # 1. Get a personal access token from GitHub (https://github.com/settings/tokens)
 #    with the following scopes enabled:
 #    * public_repo
 #    * read:org
 #    * user:email
 # 2. Set an ENV variable named 'GITHUB_SAMVERA_TOKEN' containing your token
 # 3. Then run this script:
-#    $ ruby ./script/grant_revoke_gem_authority.rb
+#    $ bundle exec ruby ./script/grant_revoke_gem_authority.rb
+#
+# By default the above script will iterate over samvera and
```

Comment:
I'm curious about what is driving this. Why not do -deprecated automatically as well? If rate-limiting concerns, ok, no problem.